src/sections/Community/Handbook/programs-data.js

```diff
@@ -40,7 +40,7 @@ export const programs_data = [
     name: "Linux Foundation",
     description:
       "The Linux Foundation (Mentorship Programs) are designed to help developers with the necessary skills–many of whom are first-time open source contributors–experiment, learn, and contribute effectively to open source communities. Layer5, as an organization, has been participating in the Linux Foundation mentorship program since 2019. ",
-    buttonLink: "/programs/lfx",
+    buttonLink: "/programs/lfx-2022/index",
```

Comment:
Only `/programs/lfx` should be the correct link.

Reply:
So should we keep the 2022 mentorship data in "lfx" and the previous year mentorship should be put into llike  folder name "lfx-2021"? In case of GSOC it also points to gsoc-2022 so that's why i thought it should be lfx-2022.

Reply:
> So should we keep the 2022 mentorship page in "lfx" and the previous year mentorship should be put into llike folder name "lfx-2021"? In case of GSOC it also points to gsoc-2022 so that's why i thought it should be lfx-2022.

Or should i create another page in lfx folder only named "2022"

Reply:
Individual pages will be created based on the changeset that you have made, so that is not an issue.
In the repository overview we should be pointing the user to the generic `/programs/lfx` page as the user will be able to switch the LFX information for that year on that itself :)

P.S: By default the latest (year) program information is shown on the page.

Reply:
yes, makes sense what @Nikhil-Ladha says . @deveshXm you can safely change it  :+1: